src/Uno.UI/UI/Xaml/Controls/Popup/PopupPanel.cs

```diff
@@ -106,6 +106,23 @@ protected override Size ArrangeOverride(Size finalSize)
 				//		 (And actually it also lets the view appear out of the window ...)
 				var anchor = Popup.Anchor ?? Popup;
 				var anchorLocation = anchor.TransformToVisual(this).TransformPoint(new Point());
+
+#if __ANDROID__
+				// for android, the above line returns the absolute coordinates of anchor on the screen
+				// because the parent view of this PopupPanel is a PopupWindow and GetLocationInWindow will be (0,0)
+				// therefore, we need to make the relative adjustment
+				if (this.GetParent() is Android.Views.View view)
+				{
+					var windowLocation = Point.From(view.GetLocationInWindow);
+					var screenLocation = Point.From(view.GetLocationOnScreen);
+
+					if (windowLocation == default)
```

Comment:
Should epsilons be used here for the comparison?

Reply:
There is no need here. Since we are comparing `Point`, and not `PointF`.